<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_divide</class>
 <widget class="QDialog" name="start_divide">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>开始分割</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>152</y>
     <width>93</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>152</y>
     <width>93</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>106</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>分割后结果保存路径：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result_path_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>106</y>
     <width>210</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_btn_2">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>106</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_combo">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>13</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>image</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>video</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nii</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>type:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="data_path_text">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>59</y>
     <width>241</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>59</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>待分割数据路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_btn_1">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>59</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
